apps/frontend/src/app/5_pages/PortfolioPage/components/AssetSection/AssetSection.constants.ts

```diff
@@ -1,14 +1,14 @@
 import { contracts } from '@sovryn/contracts';
 import { getNetworkByChainId } from '@sovryn/ethers-provider';
 import { Decimal } from '@sovryn/utils';
+
 import { RSK_CHAIN_ID } from '../../../../../config/chains';
 
-export const availableTokens =
-  contracts.assets[getNetworkByChainId(RSK_CHAIN_ID)]?.map(
-    item => item.symbol,
-  ) ?? [];
+export const getAvailableTokens = (chainId: string = RSK_CHAIN_ID) =>
+  contracts.assets[getNetworkByChainId(chainId)]?.map(item => item.symbol) ??
+  [];
 
-export const initialUsdValues = availableTokens.reduce(
+export const initialUsdValues = getAvailableTokens().reduce(
```

Comment:
Shouldn't this accept `chainId` as well? Otherwise you'll always pass `RSK_CHAIN_ID`.